src/util/StringUtils.h

```diff
@@ -4,17 +4,21 @@
 
 #pragma once
 
+#include <bits/iterator_concepts.h>
```

Comment:
Do Not Include these internal headers from gcc that start with `bits`. They do not exist in the standard.

Reply:
My bad. That probably was my editor. I auto includes things for classes/functions, that I write out with auto complete.